scif/tests/test_exec.sh

```diff
@@ -6,3 +6,4 @@ echo "Testing exec of commands:"
 docker run -it "${CONTAINER_NAME}" exec hello-world-echo echo "Hello World!"
 docker run -it "${CONTAINER_NAME}" exec hello-world-script ls /
 docker run -it "${CONTAINER_NAME}" exec hello-world-env env
+docker run -it "${CONTAINER_NAME}" exec hello-world-env echo [e]OMG
```

Comment:
I'm not familiar with the test suite here, but does somewhere check that we get the expected `TACOS` from this command... or is it just ensuring the command runs and exits with success?

Reply:
I only followed the same pattern that existed, but had the same question.  It is currently, I believe, just exiting with success (and i verified the output with my eyeballs).    I will see if I can add a better check to that.  I was in the process of trying to make my tests better and similar to those in test_tests.sh, but I'm not sure those are working properly as it is.  

@vsoch how do you/CI run these tests?